src/commands/deploy/verifyAppSettings.ts

```diff
@@ -168,3 +182,21 @@ function verifyLinuxRemoteBuildSettings(context: IActionContext, remotePropertie
     return hasChanged;
 }
 
+function verifyFeatureFlags(context: IActionContext, site: ParsedSite, remoteProperties: { [propertyName: string]: string }): boolean {
+    const featureFlagString = remoteProperties[azureWebJobsFeatureFlags] || '';
+
+    // Feature flags are comma-delimited lists of beta features
+    // https://learn.microsoft.com/en-us/azure/azure-functions/functions-app-settings#azurewebjobsfeatureflags
+    const featureFlagArray = !featureFlagString ? [] : featureFlagString.split(',');
+    const enableWorkerIndexingValue = 'EnableWorkerIndexing';
+    const shouldAddSetting: boolean = !featureFlagArray.includes(enableWorkerIndexingValue);
+
+    if (shouldAddSetting) {
+        featureFlagArray.push(enableWorkerIndexingValue);
+        ext.outputChannel.appendLog(localize('addedFeatureFlag', 'Added feature flag "{0}" because it is required for the Node v4 programming model', enableWorkerIndexingValue), { resourceName: site.fullName });
+        remoteProperties[azureWebJobsFeatureFlags] = featureFlagArray.join(',');
+    }
+
+    context.telemetry.properties.addedRunFromPackage = String(shouldAddSetting);
```

Comment:
Either remove this line or change `addedRunFromPackage` to something more accurate - depending if you want to track this in telemetry